pkg/generate/config/operation.go

```diff
@@ -15,10 +15,14 @@ package config
 
 import (
 	awssdkmodel "github.com/aws/aws-sdk-go/private/model/api"
-
+	"encoding/json"
```

Comment:
This is a standard library import, so please place above the `awssdkmodel` import line and separate with an empty newline.

Reply:
done.